contracts/wallet/WalletFactory.sol

```diff
@@ -34,60 +38,161 @@ contract WalletFactory is Owned, Managed {
     address public walletImplementation;
     // The address of the ENS manager
     address public ensManager;
+    // The address of the GuardianStorage
+    address public guardianStorage;
 
     // *************** Events *************************** //
 
     event ModuleRegistryChanged(address addr);
-    event WalletImplementationChanged(address addr);
     event ENSManagerChanged(address addr);
-    event WalletCreated(address indexed _wallet, address indexed _owner);
+    event GuardianStorageChanged(address addr);
+    event WalletCreated(address indexed wallet, address indexed owner, address indexed guardian);
 
     // *************** Constructor ********************** //
 
     /**
      * @dev Default constructor.
      */
-    constructor(address _moduleRegistry, address _walletImplementation, address _ensManager) public {
+    constructor(
+        address _moduleRegistry,
+        address _walletImplementation,
+        address _ensManager,
+        address _guardianStorage
+    )
+        public
+    {
         moduleRegistry = _moduleRegistry;
         walletImplementation = _walletImplementation;
         ensManager = _ensManager;
+        guardianStorage = _guardianStorage;
     }
 
     // *************** External Functions ********************* //
 
     /**
-     * @dev Lets the manager create a wallet for an account. The wallet is initialised with a list of modules.
+     * @dev Lets the manager create a wallet for an owner account.
+     * The wallet is initialised with a list of modules and an optional ENS..
+     * The wallet is created using the CREATE opcode.
      * @param _owner The account address.
      * @param _modules The list of modules.
-     * @param _label Optional ENS label of the new wallet (e.g. franck).
+     * @param _label (Optional) ENS label of the new wallet, e.g. franck.
      */
-    function createWallet(address _owner, address[] calldata _modules, string calldata _label) external onlyManager {
-        require(_owner != address(0), "WF: owner cannot be null");
-        require(_modules.length > 0, "WF: cannot assign with less than 1 module");
-        require(ModuleRegistry(moduleRegistry).isRegisteredModule(_modules), "WF: one or more modules are not registered");
-        // create the proxy
+    function createWallet(
+        address _owner,
+        address[] calldata _modules,
+        string calldata _label
+    )
+        external
+        onlyManager
+    {
```

Comment:
I usually use 150 as well for max length. However here we have 4 methods that have the same functionality but differ in there parameter list and implementation; I personally find that it makes the code more readable if they have the same display (easier to see their differences). Since some of them are above the 150 threshold I prefer to display them all with breaking lines.